### ISSUE TITLE
Unable to get oauth url

### ISSUE THREAD

**Issue by AndreasBackx** [runelite/runelite#8136]
**Describe the bug**
When clicking on the login button, nothing happens, and it throws an exception.

**To Reproduce**
1. Click on login when you're logged out.
2. See error

**Expected behavior**
For it to redirect me to a login page or something?

**Environment (please complete the following information):**
 - OS: Linux kernel 5.0
 - RuneLite version: 1.5.16-SNAPSHOT
 - Launcher version: Unknown

**Additional context**

```java
2019-03-07 15:27:39 [pool-2-thread-1] DEBUG n.r.http.api.account.AccountClient - Built URI: https://api.runelite.net/runelite-1.5.16-SNAPSHOT/account/login?uuid=180cafd1-1480-4dfc-bf68-df4645e02e58
2019-03-07 15:27:39 [pool-2-thread-1] WARN  n.r.client.account.SessionManager - Unable to get oauth url
java.io.IOException: com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at net.runelite.http.api.account.AccountClient.login(AccountClient.java:75)
	at net.runelite.client.account.SessionManager.login(SessionManager.java:195)
	at net.runelite.client.util.RunnableExceptionLogger.run(RunnableExceptionLogger.java:41)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:226)
	at com.google.gson.Gson.fromJson(Gson.java:927)
	at com.google.gson.Gson.fromJson(Gson.java:865)
	at net.runelite.http.api.account.AccountClient.login(AccountClient.java:71)
	... 9 common frames omitted
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:385)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:215)
	... 12 common frames omitted
```

**Comment by Adam-** [comment]
We are in the middle of a release, not everything is deployed.

**Comment by AndreasBackx** [comment]
This has been happening to me for a week. I've only now got time for making an issue, it's not related to the release I assume.

**Comment by Adam-** [comment]
This specific issue is 100% related to the release. Try to reproduce it on *my* release please.

**Comment by AndreasBackx** [comment]
I will retry after the release. The 404 is probably because of the release, but there might be a different problem as I've been having this for a while.

**Comment by AndreasBackx** [comment]
You are right, it seems to have been fixed since 1.5.13 or something else fixed it. Thank you.

**Comment by Adam-** [comment]
I do not think this has ever been broken on the release, which is what this issue tracker is for.
